grub-core/boot/i386/pc/boot.S

```diff
@@ -24,11 +24,14 @@
  *  defines for the code go here
  */
 
+#define TPM 1
```

Comment:
This was intended to make it easier to upstream. The plan was to add a configure option here.